From PeterHuewe@gmx.de  Mon Oct 26 14:31:25 2009
From: Peter Huewe <PeterHuewe@gmx.de>
Date: Tue, 29 Sep 2009 01:34:14 +0200
Subject: Staging: et131x:  adding __init/__exit macros
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Cc: devel@driverdev.osuosl.org, kernel-janitors@vger.kernel.org, Jiri Kosina <trivial@kernel.org>, linux-kernel@vger.kernel.org, Alan Cox <alan@linux.intel.com>
Message-ID: <200909290134.15295.PeterHuewe@gmx.de>
Content-Disposition: inline


From: Peter Huewe <peterhuewe@gmx.de>

Trivial patch which adds the __init/__exit macros to the module_init/
module_exit functions of ./staging/et131x/et131x_initpci.c

Greg, please have a look at the small patch and either pull it through
your staging tree, or please ack' it so Jiri can pull it through the trivial tree.

Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
Acked-by: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/et131x/et131x_initpci.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/et131x/et131x_initpci.c
+++ b/drivers/staging/et131x/et131x_initpci.c
@@ -989,7 +989,7 @@ static struct pci_driver et131x_driver =
  *
  * Returns 0 on success, errno on failure (as defined in errno.h)
  */
-static int et131x_init_module(void)
+static int __init et131x_init_module(void)
 {
 	if (et131x_speed_set < PARM_SPEED_DUPLEX_MIN ||
 	    et131x_speed_set > PARM_SPEED_DUPLEX_MAX) {
@@ -1002,7 +1002,7 @@ static int et131x_init_module(void)
 /**
  * et131x_cleanup_module - The entry point called on driver cleanup
  */
-static void et131x_cleanup_module(void)
+static void __exit et131x_cleanup_module(void)
 {
 	pci_unregister_driver(&et131x_driver);
 }
